File: Beginner/1015 (Distance Between Two Points)/1015.java
	import java.io.IOException;
import java.util.Scanner;
import java.text.DecimalFormat;
import java.lang.Math;

public class Main {

    public static void main(String[] args) throws IOException {
        DecimalFormat df = new DecimalFormat("0.0000");
        Scanner ler = new Scanner(System.in);

        double x1 = ler.nextDouble();
        double y1 = ler.nextDouble();
        double x2 = ler.nextDouble();
        double y2 = ler.nextDouble();

        System.out.println(df.format((Math.sqrt(Math.pow(x1-x2,2) + Math.pow(y1 -y2, 2)))));
        
    }
 
}

File: Beginner/1015 (Distance Between Two Points)/1015.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            string[] texto = Console.ReadLine().Split(' ');
            double x1 = double.Parse(texto[0]), y1 = double.Parse(texto[1]);
            string[] texto2 = Console.ReadLine().Split(' ');
            double x2 = double.Parse(texto2[0]), y2 = double.Parse(texto2[1]);

            Console.WriteLine( string.Format("{0:0.0000}", Math.Sqrt(Math.Pow(x1 -x2,2) + Math.Pow(y1 - y2,2))));


        }
    }
}